pkg/expr/sql_command.go

```diff
@@ -70,18 +72,18 @@ func UnmarshalSQLCommand(rn *rawNode) (*SQLCommand, error) {
 
 // NeedsVars returns the variable names (refIds) that are dependencies
 // to execute the command and allows the command to fulfill the Command interface.
-func (gr *SQLCommand) NeedsVars() []string {
-	return gr.varsToQuery
+func (cmd *SQLCommand) NeedsVars() []string {
```

Comment:
this is a nitpick, but could we do the `gr` => `cmd` renaming in a separate PR?

that would allow this one to be smaller, and the "renaming" PR would be much easier to review and approve.